libs/arcade-cli/arcade_cli/org.py

```diff
@@ -0,0 +1,169 @@
+import typer
+from rich.console import Console
+
+from arcade_cli.authn import (
+    fetch_organizations,
+    fetch_projects,
+    select_default_project,
+)
+from arcade_cli.constants import (
+    PROD_COORDINATOR_HOST,
+)
+from arcade_cli.usage.command_tracker import TrackedTyper, TrackedTyperGroup
+from arcade_cli.utils import (
+    compute_base_url,
+    handle_cli_error,
+)
+
+console = Console()
+
+
+app = TrackedTyper(
+    cls=TrackedTyperGroup,
+    add_completion=False,
+    no_args_is_help=True,
+    pretty_exceptions_enable=True,
+    pretty_exceptions_show_locals=False,
+    pretty_exceptions_short=True,
+)
+
+
+@app.command("list", help="List organizations you belong to")
+def org_list(
+    host: str = typer.Option(
+        PROD_COORDINATOR_HOST,
+        "--host",
+        "-h",
+        help="The Arcade Coordinator host.",
+    ),
```

Comment:
For commands that have subcommands like `arcade org`, the convention (at least in this repo) is to support common configuration options at the top-level command instead of at the sub-level command. `arcade secret` is an example where the top-level command handles configuration options like --host.

Example:

- `arcade secret --host api.bosslevel.dev list` instead of `arcade secret list --host api.bosslevel.dev`
- `arcade org --host cloud.bosslevel.dev list` instead of `arcade org list --host cloud.bosslevel.dev`